src/js/cityDetails.js

```diff
@@ -0,0 +1,39 @@
+/* global document */
+
+const setUpDetails = (markerGroup, data) => {
+  const cityDetailsElement = document.querySelector(".city-details-popup");
+
+  // Clicking any city marker, updates HTML and makes popup visible
+  markerGroup.on("click", function (e) {
```

Comment:
Our project uses the style of anonymous functions. In JavaScript, `function` has some weird properties and historical baggage

```suggestion
  markerGroup.on("click", (e) => {
```